CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v0.10.1 (March 10, 2022)
+Fix for a bug with the use of lock in pathVenafiCertObtain function.
```

Comment:
I think we can put this description: Fix for a bug related to the use of a synchronized block in pathVenafiCertObtain function, please let me know what you think?